wimm.Secundatives.UnitTests/Extensions/OkOr_Test.cs

```diff
@@ -89,6 +89,24 @@ public async Task OkOrErrorTaskValue_None_ReturnsErrorValue()
             Assert.Equal(TestingEnum.BadThingsHappened, result.Error);
         }
 
+        [Fact] 
+        public async Task OkOrErrorTaskValue_ResultMemberExists_ReturnsInternalResult()
```

Comment:
Are you missing a test for value/none and func/some ?

Reply:
I am

Reply:
wf